cmd/simulateSam/simulateSam_test.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"github.com/vertgenlab/gonomics/sam"
+	"math/rand"
+	"os"
+	"testing"
+)
+
+func TestSimulateSam(t *testing.T) {
+	rand.Seed(1)
+	simulateSam("testdata/test.fa", "testdata/actual.sam", 100, false)
+	simulateSam("testdata/test.fa", "testdata/actual.bam", 100, true)
+
+	if !fileio.AreEqual("testdata/actual.sam", "testdata/expected.sam") {
```

Comment:
My understanding is that the output sam involves a random sampling. So how does our output sam end up being our expected every time?